packages/iov-bns/src/bnsconnection.ts

```diff
@@ -619,6 +619,10 @@ export class BnsConnection implements BcpAtomicSwapConnection {
     return nfts;
   }
 
+  public async getFeeQuote(): Promise<null> {
+    return null;
```

Comment:
We can use

```
         tokens: {
            quantity: "0",
            fractionalDigits: 9,
            tokenTicker: "IOV" as TokenTicker,
          },
```

However, not all chains that use the BnsConnection have an IOV token. So the question is: in which currency do I have to pay the fees? Let's discuss with backend team